web/src/engine/websites/TappyToonFR_e2e.ts

```diff
@@ -0,0 +1,24 @@
+import { TestFixture } from '../../../test/WebsitesFixture';
+
+const config = {
+    plugin: {
+        id: 'tappytoon-fr',
+        title: 'TappyToon (FR)'
+    },
+    container: {
+        url: 'https://www.tappytoon.com/fr/book/the-duchess-who-sees-ghosts-fr',
+        id: '10345',
+        title: 'The Duchess Who Sees Ghosts'
```

Comment:
The chapter name is french but the title is in english, shouldn't the title be in french as well?

Reply:
I just removed the [FR] at the end as junk. Should i keep it? 
Its the same for DE where i have to remove [DE] from title.

Reply:
If i make only one multilingual plugin for TappyToon and if i clean titles like we will have identical tiltes.
Perhaps i shound not clean?